packages/nitro-protocol/test/contracts/BatchOperator/batchDeposits.test.ts

```diff
@@ -16,25 +16,25 @@ const provider = getTestProvider();
 const batchOperator = setupContract(
   provider,
   BatchOperatorArtifact,
-  process.env.BATCH_OPERATOR_ADDRESS
+  process.env.BATCH_OPERATOR_ADDRESS || ''
```

Comment:
```suggestion
  process.env.BATCH_OPERATOR_ADDRESS!
```
Works too, and is possibly a bit nicer?

I think somewhere we have some type file to force these as global variables... tagging @NiloCK for input.

Reply:
I think that's going to violate the [no-null-assertion](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-non-null-assertion.md) linting rule.

Reply:
Ah ok. Not a huge deal I think.